src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+
+	}
+	
+	@FindBy(how = How.XPATH, using = "//*[@id='order-18257']/td[1]")
+	//*[@id="order-18257"]/td[1]
+	//*[@id="18258"]/span
+	private WebElement clickOrder;
+
+	/**
+	 * Method to click on Save Changes Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickOrder() throws IOException {
+		OrderHierarchiesPage sp = new OrderHierarchiesPage(driver);
+		log.info("Click on Save Changes Button");
+		Assert.assertTrue(clickOrder.isDisplayed());
+		clickOrder.click();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//a[@class='submit order']//*[text()='Generate Invoice']")
+	private WebElement clickGenerateInvoiceButton;
+
+	/**
+	 * Method to Click on Generate Invoice Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickGenerateInvoiceButton() throws IOException {
+		JavaScriptExec.scrollToElementOnPage(driver, clickGenerateInvoiceButton);
+		GeneratingInvoicePage sp = new GeneratingInvoicePage(driver);
+		log.info("Click on Generate Invoice Button");
+		Assert.assertTrue(clickGenerateInvoiceButton.isDisplayed());
+		clickGenerateInvoiceButton.click();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//div[@class='msg-box successfully']//*[text()='Done']")
+	private WebElement verifyConfirmationMsg;
+
+	/**
+	 * Method to verify Account Type is created Successfully.
+	 * 
+	 * @throws IOException
+	 */
+	public void verifyConfirmationMsg() throws IOException {
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		log.info("Verifying if Account Type is created Successfully or not");
+		JavaScriptExec.sleep();
+		Assert.assertTrue(verifyConfirmationMsg.isDisplayed(),
+				"Assert Failed as its unable to search text in Logged in Page");
+	}
+	
+	
+ 		@FindBy(how = How.XPATH, using = "//button[@id='newSubOrder']")
+ 		
+ 	private WebElement clickNewSubOrderButton;
+
+ 		/**
+ 		 * Method to click on New Sub Order Order button.
+ 		 * 
+ 		 * @throws IOException
+ 		 */
+ 	public void clickNewSubOrderButton() throws IOException {
+ 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 		log.info("click on New Sub Order button.");
+ 		Assert.assertTrue(clickNewSubOrderButton.isDisplayed());
+ 		clickNewSubOrderButton.click();
+ 		JavaScriptExec.sleep();
+ 		}
+ 		
+ 	   @FindBy(how=How.XPATH, using="//*[@id='ui-id-6']")
+ 		//*[@id="ui-id-6"]
+ 	//*[@id="ui-id-6"]
+ 	 private WebElement clickDetailsTab;
+ 	    /**
+ 	     * Method to Click Products tab.
+ 	     * @throws IOException 
+ 	     */
+ 	  public void clickDetailsTab() throws IOException{
+ 	    CreateOrderPage sp=new CreateOrderPage(driver);
+ 	    log.info("Click Details Tab");
+ 	    Assert.assertTrue(clickDetailsTab.isDisplayed());
+ 	    clickDetailsTab.click();
+ 	    JavaScriptExec.sleep();
+ 	    }
+ 	    
+ 	 @FindBy(how=How.XPATH, using="//*[@id='order-details-form']/div[1]/div[1]/span/a")
+ 	//*[@id="order-details-form"]/div[1]/div[1]/span/a
+	 private WebElement clickNew;
+	    /**
+	     * Method to Click Products tab.
+	     * @throws IOException 
```

Comment:
@mayankchhajer Please remove the xpaths after pasting

Reply:
Done